config/base/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
 namespace: vm
-namePrefix: vm-
```

Comment:
because it looks better:

```
$kubectl -n vm get pod 
NAME                             READY   STATUS              RESTARTS   AGE
vmagent-demo-5855fddf5d-wj54g    2/2     Running             0          14s
vmalert-demo-87c78595d-nqmgb     0/2     ContainerCreating   0          4s
vmalertmanager-demo-0            0/2     ContainerCreating   0          6s
vmoperator-585f4d9b7-fsft4       1/1     Running             0          34s
vmsingle-demo-744b9c44cb-5b567   1/1     Running             0          21s
```